frb/scripts/frb_summary.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""
+Script to print a summary of a given FRB to the terminal
+"""
+from IPython import embed
+
+def parser(options=None):
+    import argparse
+    # Parse
+    parser = argparse.ArgumentParser(description='Script to print a summary of an FRB to the screen [v1.0]')
+    parser.add_argument("frb_name", type=str, help="FRB name, e.g. FRB180924 or simply 180924")
+    parser.add_argument("--verbose", default=False, action="store_true", help="Overwhelm the screen?")
+
```

Comment:
One useful option I can think of is to show the image of the host with the FRB position. I'd be happy to add this functionality if you think that's a good idea.

Reply:
I think it is a fine idea.  I suggest you wait until `associate` clears
which has in it pointers to the 'best' images for each FRB:

Reply:
https://github.com/FRBs/FRB/blob/associate/frb/associate/frbs.py